packages/accounts/plugindefs/index.ts

```diff
@@ -1,3 +1,7 @@
 export { MultiOwnerPluginGenConfig } from "./multi-owner/config.js";
 export { SessionKeyPluginGenConfig } from "./session-key/config.js";
 export { TokenReceiverPluginGenConfig } from "./token-receiver/config.js";
+
+export { MultiOwnerPluginAbi } from "./multi-owner/abi.js";
+export { SessionKeyPluginAbi } from "./session-key/abi.js";
+export { TokenReceiverPluginAbi } from "./token-receiver/abi.js";
```

Comment:
instead of exporting and importing these (because this folder is not outputted in the final bundle), the Abi for the contract is available during the plugin gen phase. You can use it there to populate the `getContract` I think

Reply:
https://github.com/alchemyplatform/aa-sdk/pull/358/files#diff-29103509588947d27bb0a950af40aa9d8eace89db6ef63d3927cee22e466dab7R80 this is where we use the contract's abi to do other stuff, you could just dump it into a private variable in the generated code

Reply:
I tried using that, but that did not work because contract.abi is viem's `Abi` typed.

Reply:
Should work since you can do`JSON.stringify(contract.abi)` in the code